tests/transformer/test_operators.py

```diff
@@ -0,0 +1,356 @@
+def test_Eq(c_eval):
+    result = c_eval('1 == 1')
+    assert result.code is not None
+    assert result.errors == ()
+    assert result.warnings == []
+    assert result.used_names == {}
+
+    eval_result = eval(result.code)
+    assert eval_result is True
+
+
+@pytest.mark.parametrize(*c_eval)
+def test_NotEq(c_eval):
+    result = c_eval('1 != 2')
+    assert result.code is not None
+    assert result.errors == ()
+    assert result.warnings == []
+    assert result.used_names == {}
+
+    eval_result = eval(result.code)
+    assert eval_result is True
+
+
+@pytest.mark.parametrize(*c_eval)
+def test_Gt(c_eval):
+    result = c_eval('2 > 1')
+    assert result.code is not None
+    assert result.errors == ()
+    assert result.warnings == []
+    assert result.used_names == {}
+
+    eval_result = eval(result.code)
+    assert eval_result is True
+
+
+@pytest.mark.parametrize(*c_eval)
+def test_Lt(c_eval):
+    result = c_eval('1 < 2')
+    assert result.code is not None
+    assert result.errors == ()
+    assert result.warnings == []
+    assert result.used_names == {}
+
+    eval_result = eval(result.code)
+    assert eval_result is True
+
+
+@pytest.mark.parametrize(*c_eval)
+def test_GtE(c_eval):
+    result = c_eval('2 >= 2')
+    assert result.code is not None
+    assert result.errors == ()
+    assert result.warnings == []
+    assert result.used_names == {}
+
+    eval_result = eval(result.code)
+    assert eval_result is True
+
+
+@pytest.mark.parametrize(*c_eval)
+def test_LtE(c_eval):
+    result = c_eval('1 <= 2')
+    assert result.code is not None
+    assert result.errors == ()
+    assert result.warnings == []
+    assert result.used_names == {}
+
+    eval_result = eval(result.code)
+    assert eval_result is True
+
+
+# Bool Operators
+
+
+@pytest.mark.parametrize(*c_eval)
+def test_Or(c_eval):
+    result = c_eval('False or True')
+    assert result.code is not None
+    assert result.errors == ()
+    assert result.warnings == []
+    # assert result.used_names == {}
+
+    eval_result = eval(result.code)
+    assert eval_result is True
+
+
+@pytest.mark.parametrize(*c_eval)
+def test_And(c_eval):
+    result = c_eval('True and True')
+    assert result.code is not None
+    assert result.errors == ()
+    assert result.warnings == []
+    # assert result.used_names == {}
+
+    eval_result = eval(result.code)
+    assert eval_result is True
+
+
+@pytest.mark.parametrize(*c_eval)
+def test_Not(c_eval):
+    result = c_eval('not False')
+    assert result.code is not None
+    assert result.errors == ()
+    assert result.warnings == []
+    # assert result.used_names == {}
+
+    eval_result = eval(result.code)
+    assert eval_result is True
+
+
+# Bit wise Operators
+
+
+@pytest.mark.parametrize(*c_eval)
+def test_BitAnd(c_eval):
+    result = c_eval('60 & 13')
+    assert result.code is not None
+    assert result.errors == ()
+    assert result.warnings == []
+    assert result.used_names == {}
```

Comment:
Huh, no assertion of the executed result?
Maybe `5 & 3` would be easier to understand as example.